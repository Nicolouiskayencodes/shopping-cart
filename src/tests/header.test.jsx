import { describe, it, expect } from "vitest";
import { render } from "@testing-library/react";
import Header from "../components/header";
import { BrowserRouter } from "react-router-dom";

describe("Header component", () => {
  it("renders Header", ()=>{
    const { container } = render(<BrowserRouter><Header itemsTotal={2}/></BrowserRouter>);
    expect(container).toMatchSnapshot();
  })
});