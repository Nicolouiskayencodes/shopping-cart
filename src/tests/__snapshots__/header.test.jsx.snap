// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Header component > renders Header 1`] = `
<div>
  <div>
    <h1>
      InterPorium
    </h1>
    <a
      href="/"
    >
      Home
    </a>
    <hr />
    <a
      href="/shopping"
    >
      Shop
    </a>
    <hr />
    <a
      href="/cart"
    >
      <img
        src="/src/assets/cart-variant.svg"
      />
       
      2
    </a>
  </div>
</div>
`;
